### DISCUSSION TITLE
Is there a way to separate lead vocals and back up vocals?

### DISCUSSION THREAD

**Discussion by Benji0TheFox** [Anjok07/ultimatevocalremovergui#1250] [Q&A]
Lets say there is one stem that has both the lead and the backup, is it possible to split the lead and the backup into separate vocal tracks?

**Comment by saulop** [comment]
Go to Settings, tab "additional settings".  Then button "Vocal Splitter Options". You can then enable vocal split mode and select a model such as MDX Karaoke 2.

**Reply by flonou** [reply]
This works well but it doesn't seem to extract backing vocals and main vocals separately as stated in the options. I only get instrumental + backing vocals in one file and main vocals in the other

**Reply by RevEng** [reply]
The "Save Split Vocal Instrumentals" setting doesn't seem to have any effect for me neither (Ultimate Vocal Remover v5.6.0 [2024-06-06 23:18:53]).

If you want the two vocals separate, you can use the same models, but you'll need to run it twice without the vocal split mode.

Run the original file through the UVR-MDX-NET Karaoke 2 model and save both the Vocals and Instrumentals. This first Vocals contains both vocal parts.

Run the Vocals through the same model and save both Vocals and Instrumentals. This second Vocals will be your lead vocals and the second Instrumentals will be the backing vocals.

This is exactly the same process that the Vocal Split option does, only it combines the backing vocals with the instrumental afterwards, making a perfect karaoke track with only the lead vocal removed.

**Reply by bishopnerth** [reply]
> This works well but it doesn't seem to extract backing vocals and main vocals separately as stated in the options. I only get instrumental + backing vocals in one file and main vocals in the other

Please try this with the settings given; do the first splitting, take the splitted audio through the settings just given (already, there won't be instruments in the splitted aufio), so I think it should work. All the best, boss

**Comment by sdflemming** [comment]
Under Ensemble Mode :

Kim Vocal 1 + Karaoke 1 + Karaoke 2 

to try and separate lead vocal from background vocals

I've had a great result running the audio thru Kim Vocals 2 them taking the exported audio and running again using HP-Karaoke-UVR. It cleaned the minor artifacts the Kim Vocals 2 model left.

**Comment by Th3w33knd** [comment]
https://github.com/Anjok07/ultimatevocalremovergui/discussions/689#discussioncomment-7459819 <- I like to do it this way. There "chorus" is your "backup". Karaoke models are the one which splits your "Lead + Backup" into "Lead" and "Backup"

**Comment by dinhduongson** [comment]
use MDX-net model Kim Vocal 2 and select option Vocals Only

**Comment by SilSinn9801** [comment]
I saw [this Reddit post](https://www.reddit.com/r/IsolatedTracks/comments/1axq7ky/how_to_separate_main_vocals_from_backing_vocals/) which suggested the following:

1. Input: original track > Method: MDX-Net > Model: Kim Vocal 2 > Output: Vocals contains both lead & backing vocals, Instrumental contains all instruments.
2. Input: Vocals output from (1.) above > Method: VR Architecture > Model: 5_HP-Karaoke-UVR > Output: Instrumental contains backing vocals, Vocals contains lead vocals.

I think the same (or better) could be achieved using 6_HP-Karaoke-UVR unless someone points me out an important issue with that particular model.
